### ISSUE TITLE
Enable navigating in changelog message

### ISSUE THREAD

**Issue by djschilling** [churchtools/changelogger#21]
When writing the changelog message i cannot use the keyboard navigation.

The back key leads to this character: `^[[D`.

On the mac i would like to use the back and forward keys and also the combination with `alt` to navigate through words.

**Comment by obstschale** [comment]
It seems the console package used to supported that but removed it because it was buggy. I opend an issue to ask if this is still the case or it might be possible now. https://github.com/symfony/symfony/issues/36657
